### ISSUE TITLE
Make the CLI work on CI

### ISSUE THREAD

**Issue by egoist** [Dokploy/cli#2]
I want to use the CLI on CI to automate deployment, I know there's API available but why not make the CLI capable of running on CI?

For example `dokploy app deploy` uses inquirer to get application id interactively, ideally it should support `dokploy app deploy --appId myId`

For this to work it should also read auth info from environment variables:

```
DOKPLOY_URL=xxx DOKPLOY_AUTH_TOKEN=xxx dokploy app deploy --appId myId
```

**Comment by Siumauricio** [comment]
Yeah makes sense, I have to adapt the commands to accept the inline flags
